### PR TITLE
Tablet designs & UI Search implementation

### PR DESCRIPTION
**Pull request by Brioline**
UI: Search implementation - https://github.com/polywrap/hub/issues/101

Contains:
* https://github.com/polywrap/hub/issues/101
* https://github.com/polywrap/hub/issues/91
* https://github.com/polywrap/hub/issues/107
* https://github.com/polywrap/hub/issues/103

### CODE REVIEW

File: src/hooks/useCreateApi.tsx [hunk 1]
```diff
@@ -5,12 +5,12 @@ import React, {
   useContext,
 } from "react";
 
-type UseCreteApiContextValue = {
+type ResponsiveContextValue = {
```
Comment (cbrzn) [inline src/hooks/useCreateApi.tsx:8]: Why is this being renamed to ResponsiveContext?

File: src/hooks/useResponsive.tsx [hunk 2]
```diff
@@ -34,6 +33,6 @@ export const ResponsiveProvider = ({
   );
 };
 
-export const useResponsive = () => useContext(ResponsiveContext);
```
Comment (cbrzn) [inline src/hooks/useResponsive.tsx:37]: What's the reason for removing this?

**Review by cbrzn — COMMENTED** [review]
Looks amazing! Just left a few comments

File: src/hooks/useResponsive.tsx (hunk 2 above)
Comment (Brioline) [reply src/hooks/useResponsive.tsx:37]: There is a default export for this

File: src/hooks/useCreateApi.tsx (hunk 1 above)
Comment (Brioline) [reply src/hooks/useCreateApi.tsx:8]: Copy/Paste mistake. Solved with recent commit

**Comment by Brioline** [comment]
@cbrzn could you please comment on usage of Web3Api client ? Is this how it supposed to work ? Especially I'm concerned with making 4-5 requests to all gateways when we try to fetch any query of a package

**Pull request merged**